loader/base_loader.py

```diff
@@ -104,6 +112,24 @@ def __init__(self, dss_endpoint: str, staging_bucket: str, google_project_id: st
         dss_config['DSSClient'].swagger_url = f'{self.dss_endpoint}/swagger.json'
         self.dss_client = DSSClient(config=dss_config)
 
+    def mk_s3_metadata_client(self, aws_meta_cred):
+        """Access AWS credentials from a file and supply a client for them."""
+        if not aws_meta_cred:
+            return None
+
+        with open(aws_meta_cred, 'r') as f:
+            access_key, secret_key = [i.strip() for i in f.read().split(',')]
+        return boto3.client('s3', aws_access_key_id=access_key, aws_secret_access_key=secret_key)
+
+    def mk_gs_metadata_client(self, gce_meta_cred):
+        """Access Google credentials from a file and supply a client for them."""
+        if not gce_meta_cred:
+            return None
+
+        with open(gce_meta_cred, 'r') as f:
```

Comment:
Rather than read credentials file directly, I think you must read it using a Google Credentials class. Specifically, I think google.oauth2.credentials.Credentials#from_authorized_user_file
 is what you want to use.